environment-management/set-warning-off.sh

```diff
@@ -0,0 +1,25 @@
+SHELL_NAME=$(basename $SHELL)
```

Comment:
What are these two files doing?

Reply:
So it basically writes the 'export blah=blah' assignment to your profile of choice. but offering npm run develop/npm run edit accomplishes the same thing so it's kind of duplicated work.